src/components/skeletons/SkeletonElement/skeleton-element.scss

```diff
@@ -0,0 +1,34 @@
+[data-store-skeleton-element] {
+  &[data-shimmer="true"] {
+    position: relative;
+    overflow: hidden;
```

Comment:
Does this prop is really necessary, depending what are you rendering, the browser will need to work twice. 
1. render and get rect for this element
2. calculate the overflow rect to paint.

Reply:
This prop is necessary to make the `SkeletonElement` work independently and to have the shimmer effect, without needing a parent wrapper component with relative position

Reply:
Ok, np